ui-frontend/packages/catalog-ui-search/src/main/webapp/react-component/results-export/container.tsx

```diff
@@ -147,7 +147,20 @@ class ResultsExport extends React.Component<Props, State> {
         count,
       },
     ]
-
+    const currentAttrs = user
+      .get('user')
+      .get('preferences')
+      .get('inspector-summaryShown')
+    const columnOrder = currentAttrs.concat(
```

Comment:
❓ I may be misunderstanding the new desired behaviour, but I thought we only wanted to include the "shown"/"active" attributes, so just the `currentAttrs` here?

Reply:
Agreed - using the "Active" attributes per the "Manage Attributes" view gives the user the most configurability. Updating to only pull in the 'inspector-summaryShown" attributes.